### DISCUSSION TITLE
Is GPT-4 supported by revChatGPT.V1?

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [acheong08/ChatGPT#1197] [Q&A]
Is GPT-4 supported by `revChatGPT.V1`?

**Comment by acheong08** [comment]
https://github.com/acheong08/ChatGPT#--optional-configuration

Yes

**Reply by acheong08** [reply]
Only plus accounts can use it

**Reply by hongyi-zhao** [reply]
Do you mean that the [waitlist](https://openai.com/waitlist/gpt-4-api) based method is in vain?

**Reply by acheong08** [reply]
Waitlist allows you to pay for the API. You can already use `chat.openai.com` with gpt-4 with a plus account since the day it was released

**Reply by hongyi-zhao** [reply]
Do you mean that the free quota, aka, `$18`, cannot be used for this purpose?

![image](https://user-images.githubusercontent.com/11155854/226575615-75deebe8-245f-4d7b-a83b-a936a3c41c8d.png)

**Reply by acheong08** [reply]
Not unless you get into the waitlist

**Reply by acheong08** [reply]
But you can once you successfully apply

**Reply by hongyi-zhao** [reply]
My current status is as follows:

![image](https://user-images.githubusercontent.com/11155854/226575887-f9631729-cec5-43eb-9bf3-2f642108d6e2.png)

But I don't know when I can get approval to join the waiting list. How long does it take in your experience?

**Reply by acheong08** [reply]
Depends. The other big names got access within hours @transitive-bullshit and @waylaidwanderer. I still don't have it

**Reply by hongyi-zhao** [reply]
> Depends. The other big names got access within hours @transitive-bullshit and @waylaidwanderer.

How do you know this information?

**Reply by waylaidwanderer** [reply]
We have a discord server that we chat in @hongyi-zhao :) You can find an invite link here: https://www.chatgpthackers.dev/

**Reply by hongyi-zhao** [reply]
Nice. I've joined it.
